### PR TITLE
Split combined backup flow to multiple pages and cleanup interface design

### PR DESCRIPTION
**Pull request by beansgum**
This pr adds a new package `seedbackup`  and splits the seed backup page into backup instructions page, save seed page, verify seed page, and backup success page. The user interface is also cleaned up to apply proper styling and dimensions.

### CODE REVIEW

File: ui/page/seedbackup/backup_success.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+func (pg *BackupSuccessPage) Handle() {
+	for pg.actionButton.Clicked() {
+		pg.PopToFragment(components.WalletsPageID)
+	}
+}
+
+func (pg *BackupSuccessPage) OnClose() {}
+
+// - Layout
```
Comment (oshorefueled) [inline ui/page/seedbackup/backup_success.go:48]: was this added here intentionally?

File: ui/page/seedbackup/save_seed.go [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+					pg.actionButton.Color = pg.Theme.Color.InvText
+
+					return layout.Inset{Top: values.MarginPadding16}.Layout(gtx, pg.actionButton.Layout)
+				}))
+		}),
+	)
+}
+
+func (pg *SaveSeedPage) seedRow(gtx C, row saveSeedRow) D {
+	itemWidth := gtx.Constraints.Max.X / 3
```
Comment (oshorefueled) [inline ui/page/seedbackup/save_seed.go]: could adding a comment here make the value 3 look less like a magic number?

**Comment by oshorefueled** [comment]
There should probably be a little bit of space between the fixed bottom pane and the side navigation similar to how it's done on the send page.

###### Current Seed backup implementation 
Screenshot 2021-09-01 at 21 28 10

###### Send page mockup
Screenshot 2021-09-01 at 20 54 48

File: ui/page/seedbackup/backup_success.go (hunk 1 above)
Comment (beansgum) [reply ui/page/seedbackup/backup_success.go:48]: Yes

**Review by oshorefueled — DISMISSED** [review]

File: ui/page/components/sub_page.go [hunk 3]
```diff
@@ -58,15 +58,23 @@ func (sp *SubPage) Header(gtx layout.Context) layout.Dimensions {
...
-			}
+			title := sp.Load.Theme.Label(values.TextSize20, sp.Title)
+			title.Color = sp.Load.Theme.Color.DeepBlue
+
 			return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
-				layout.Rigid(sp.Theme.H6(sp.Title).Layout),
-				layout.Rigid(sp.Theme.Body1(sp.SubTitle).Layout),
+				layout.Rigid(title.Layout),
+				layout.Rigid(func(gtx C) D {
+					if sp.SubTitle == "" {
```
Comment (Sirmorrison) [inline ui/page/components/sub_page.go]:
```suggestion
					if components.StringNotEmpty(sp.SubTitle) {
```

File: ui/page/seedbackup/backup_instructions.go [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+				Height:      decredmaterial.WrapContent,
+				Orientation: layout.Vertical,
+				Direction:   layout.S,
+				Alignment:   layout.Baseline,
+				Background:  theme.Color.Surface,
+				Padding:     layout.UniformInset(values.MarginPadding16),
+				Margin:      layout.Inset{Left: values.Size0_5},
+			}.Layout(gtx,
+				layout.Rigid(func(gtx C) D {
+					if infoText == "" {
```
Comment (Sirmorrison) [inline ui/page/seedbackup/backup_instructions.go]:
```suggestion
					if components.StringNotEmpty(infoText) {
```

File: ui/page/seedbackup/backup_instructions.go [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+						return D{}
+					}
+					label := theme.Label(values.TextSize14, infoText)
+					label.Color = theme.Color.Gray3
+					return layout.Inset{Bottom: values.MarginPadding16}.Layout(gtx, label.Layout)
+				}),
+				layout.Rigid(func(gtx C) D {
+					gtx.Constraints.Min.X = gtx.Constraints.Max.X
+
```
Comment (Sirmorrison) [inline ui/page/seedbackup/backup_instructions.go]: Remove white space

File: ui/page/seedbackup/backup_success.go [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+	return pg
+}
+
+func (pg *BackupSuccessPage) ID() string {
+	return BackupSuccessPageID
+}
+
+func (pg *BackupSuccessPage) OnResume() {
+
+}
```
Comment (Sirmorrison) [inline ui/page/seedbackup/backup_success.go]:
```suggestion
func (pg *BackupSuccessPage) OnResume() {}
```

File: ui/page/seedbackup/verify_seed.go [hunk 7]
```diff
@@ -0,0 +1,282 @@
...
+		shuffledSeed.clickables = append(shuffledSeed.clickables, &widget.Clickable{})
+		allSeeds = removeSeed(allSeeds, randomSeed)
+	}
+
+	rand.Shuffle(len(shuffledSeed.words), func(i, j int) {
+		shuffledSeed.words[i], shuffledSeed.words[j] = shuffledSeed.words[j], shuffledSeed.words[i]
+	})
+
+	return shuffledSeed
+
```
Comment (Sirmorrison) [inline ui/page/seedbackup/verify_seed.go]: Remove whitespace

File: ui/page/seedbackup/verify_seed.go [hunk 8]
```diff
@@ -0,0 +1,282 @@
...
+			}
+		}
+	}
+
+	for pg.actionButton.Clicked() {
+		if pg.allSeedsSelected() {
+			pg.verifySeed()
+		}
+	}
+
```
Comment (Sirmorrison) [inline ui/page/seedbackup/verify_seed.go]: Remove whitespace

**Review by Sirmorrison — CHANGES_REQUESTED** [review]
Since this is a clean up PR,

When the page is expanded, the verify button is overly large in width.
Following recent review by Raedah, the button should have a maximum with which is preferably the width of the button on the default screen.

image

this button size is perfect
image

- the button color when it is grayed out does not correspond to the other buttons on the app.
the color should be `t.Color.InactiveGray = rgb(0xc4cbd2)`

- On the Write down seed phrase page, the confirm button is also overly large on full screen. it should have a maximum width and center aligned with the text on full screen.
image

- On the seed phrase page, the action button width is overly expanded also and the color is not uniform to the application (see recommended color above.
- Then i noticed a pattern with the seed phrase arrangement. it does not seem to be random as it has a pattern. The first 10 are on thesame line, the next 10 on the next line and so on...
image

image

image

- Clicking on the back button on the back button on the `Verify seed phrase page 2/2` page takes me back to the `wallet page` instead of `page 1/2`.

File: ui/page/components/sub_page.go [hunk 9]
```diff
@@ -32,11 +32,11 @@ func SubpageHeaderButtons(l *load.Load) (decredmaterial.IconButton, decredmateri
 	backButton := l.Theme.PlainIconButton(new(widget.Clickable), l.Icons.NavigationArrowBack)
 	infoButton := l.Theme.PlainIconButton(new(widget.Clickable), l.Icons.ActionInfo)
 
-	zeroInset := layout.UniformInset(values.MarginPadding0)
+	zeroInset := layout.UniformInset(values.MarginPadding4)
```
Comment (Sirmorrison) [inline ui/page/components/sub_page.go]: Please rename as the value is not longer `values.MarginPadding0`

**Review by Sirmorrison — CHANGES_REQUESTED** [review]
- Clicking on the back button on the back button on the Verify seed phrase page 2/2 page takes me back to the wallet page instead of page 1/2. (see screen recording below.)
ezgif com-gif-maker(4)

- furthermore, this issues is only noticed on windows machine for me.. On Mac it seems to work fine.
image

**Review by Sirmorrison — APPROVED** [review]

**Review by dreacot — APPROVED** [review]

**Pull request merged**